app/models/submission.rb

```diff
@@ -1,4 +1,22 @@
 class Submission < ActiveRecord::Base
-  has_one :person
-  has_one :repo
+  belongs_to :student
+  belongs_to :repo
+
+  def pr_id
+    self.pr_url.split('/').last
+  end
+
+  def type
+    if self.pr_url && !self.feedback_url
+      return 'warning'
+    elsif self.pr_url && self.feedback_url
+      return 'success'
+    else
+      return 'danger'
+    end
```

Comment:
Could we rewrite this as:
```ruby
if !self.pr_url
  'danger'
elsif !self.feedback_url
  'warning'
else
  'success'
end
```

Also, since we're only using this for determining the class to use in our view logic we should probably make this a view helper method instead.